names/names.py

```diff
@@ -16,7 +16,8 @@
         if name_1 == name_2:
             duplicates.append(name_1)
 
-end_time = time.time()
-print (f"{len(duplicates)} duplicates:\n\n{', '.join(duplicates)}\n\n")
-print (f"runtime: {end_time - start_time} seconds")
+duplicates = set(names_1) & set(names_2)
```

Comment:
Lol, amazing job here Jonas. This is by far one of the best implementations of a stretch I've seen. Great job!